platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/isolated/AbstractIsolatedProjectsToolingApiIntegrationTest.groovy

```diff
@@ -33,4 +33,10 @@ class AbstractIsolatedProjectsToolingApiIntegrationTest extends AbstractIsolated
     GradleExecuter createExecuter() {
         return new ToolingApiBackedGradleExecuter(distribution, temporaryFolder)
     }
+
+    def createProjectDirs(String... names) {
```

Comment:
I don't think the name of this method reflects what it does.

Based on the name I would only expect it to create directories, but it also includes them in the settings file.

Reply:
how about `setupProjectDirs`?

Reply:
I think the more significant part is it includes the projects as part of the build. IMO the fact that there needs to be a dir at all is a side effect of the project being included.

I would just call it `includeProjects`